sentry/src/main/java/io/sentry/FullyDisplayedReporter.java

```diff
@@ -0,0 +1,42 @@
+package io.sentry;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public final class FullyDisplayedReporter {
```

Comment:
```suggestion
public final class FullDisplayedReporter {
```
`l`: for the sake of aligning